### ISSUE TITLE
Cross chunk protections should apply to other deposit kinds.

### ISSUE THREAD

**Issue by Rosethorns** [oitsjustjose/Geolosys#286]
## Versions:

- Minecraft Forge: 38.0.15
- Geolosys: 1.18
- JourneyMap (optional):
- ImmersiveEngineering (optional):

## What happens: chunk holes in deposit_layer style deposits
![image](https://user-images.githubusercontent.com/2171914/145519409-98a7a6e6-c436-469a-bbdb-bb9c8d27af6a.png)

## What should happen:
Should form deposits uniformly across chunks as it does with deposit_sparse

## Additional Comments:
While the risk of cascading worldgen is a possibility, a deposits size is predictable and cascading worldgen therefore manageable.

**Comment by oitsjustjose** [comment]
All plutons (in fact, even samples) use cross-chunk protection. I think I have fixed this issue, but do let me know if you in fact continue to have this issue in `7.0.0-alpha2`. Below is a screenshot where the chunk I'm standing in is where the pluton started. I set it up to use a huge radius (`19`) and happened across this one after some heavy refactoring.

![image](https://user-images.githubusercontent.com/3887324/145921663-8ba6a0c9-2812-4fbd-8541-40f32daa4040.png)
